mlflow/store/db_migrations/versions/a1b2c3d4e5f6_add_spans_table.py

```diff
@@ -81,4 +150,12 @@ def upgrade():
 
 
 def downgrade():
+    bind = op.get_bind()
+
+    if bind.dialect.name == "postgresql":
+        from sqlalchemy import text
```

Comment:
Can we dedupe / move this import to the top and remove the unnecessary newlines?